pkg/defaults/defaults.go

```diff
@@ -26,6 +26,9 @@ const (
 	// GopsPortApiserver is the default value for option.GopsPort in the apiserver
 	GopsPortApiserver = 9892
 
+	// GopsPortKVStoreMesh is the default value for option.GopsPort in kvstoremesh
+	GopsPortKVStoreMesh = 9893
```

Comment:
This value is already used for Hubble Relay's gops port:

https://github.com/cilium/cilium/blob/68b3ae1918b1bd81e2f82d4fb83c896f8fae4f9f/Documentation/operations/system_requirements.rst?plain=1#L431

https://github.com/cilium/cilium/blob/68b3ae1918b1bd81e2f82d4fb83c896f8fae4f9f/pkg/hubble/relay/defaults/defaults.go#L20-L21

I guess this should be 9894 instead.

Would also be nice to document the new value in the table:

https://github.com/cilium/cilium/blob/68b3ae1918b1bd81e2f82d4fb83c896f8fae4f9f/Documentation/operations/system_requirements.rst?plain=1#L413-L436

Reply:
KVStoreMesh is not running in host network, hence it does not risk conflicting with the ports used on the nodes. Do you think it is better to add it to the system requirements also in that case? (I see that clustermesh-apiserver is present there, although it's unclear to me whether that is intentional or due to historical reasons).

Reply:
I'd still suggest to keep the port values distinct for consistency.

Feel free to omit it from the table, that one admittedly is a bit misleading probably for historical reasons and could use a cleanup.

Reply:
I've changed the port value to 9894 and opened https://github.com/cilium/cilium/pull/26230 for the cleanup. PTAL